### ISSUE TITLE
Cannot connect to server via VSCode [BUG]

### ISSUE THREAD

**Issue by ioannidesalex** [hestiacp/hestiacp#1647]
**Describe the bug**
mkdir: cannot create directory ‘/home/user/.vscode-server’: Permission denied
Creating the server install dir failed...

**To Reproduce**
What steps did you take when the issue occured? 
Login to server via VSCode

**Expected behavior**
The folder to be created

**Screenshots**

**Operating system:**
Ubuntu 20.04

**Hestia Control Panel version:**
1.3.2

**Additional context**
Basically we need to add a directory like we do with .composer etc, since jail doesn't allow creations in home folder without admin priviledges.

Something like:

mkdir -p /home/$user/.vscode-server
chown $user:$user /home/$user/.vscode-server
chmod 755 /home/$user/.vscode-server

**Comment by jaapmarcus** [comment]
Changed
